spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisQueueMessageDrivenEndpoint.java

```diff
@@ -80,6 +80,8 @@
 
 	private volatile Runnable stopCallback;
 
+	private volatile boolean readFromLeft = false;
```

Comment:
`pop-right="true/false"` on inbound and `push-left="true/false"` on outbound ? :smile: 
I don't like to introduce one more `enum` for `RIGHT/LEFT` constants...